config/manifests/bases/metallb-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: metallb-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: An operator for deploying MetalLB on a kubernetes cluster.
+  displayName: Metallb Operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - metallb-operator
+  links:
+  - name: Metallb Operator
+    url: https://github.com/metallb/metallb-operator
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat
+  maturity: alpha
+  provider:
+    name: Red Hat
```

Comment:
same

Reply:
@russellb should i also put you as the provider?

Reply:
the docs say `The Operators' provider (usually an organization), with a name.`

how about "MetalLB Project" ?